### PR TITLE
XW-2203 | using DS variable for body font family

### PR DESCRIPTION
**Pull request by weronika**
* using $wds-font-family for defining body font family

https://wikia-inc.atlassian.net/browse/XW-2203

### CODE REVIEW

**Comment by kvas-damian** [comment]
I'm not 100% sure if we should change it, but I found `font-family definition in https://github.com/Wikia/mercury/blob/dev/server/app/views/_layouts/error.hbs#L11

What do you think?

**Comment by kvas-damian** [comment]
The same question about auth pages: https://www.wikia.com/signin

**Comment by weronika** [comment]
We decided not to change font-family in https://github.com/Wikia/mercury/blob/dev/server/app/views/_layouts/error.hbs#L11, as styling there is defined in head, so it's not possible to use DS variable.

**Comment by rogatty** [comment]
@weronika please just copy the variable's value to `error.hbs`. Chinese errors will look better :smiley:

File: front/main/app/styles/component/design-system/_wikia-styleguide-overrides.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
+body {
+	font-family: $wds-font-family;
```
Comment (rogatty) [inline front/main/app/styles/component/design-system/_wikia-styleguide-overrides.scss]: I don't think it belongs here. We don't override DS styles but use a variable from it.
How about https://github.com/Wikia/mercury/blob/dev/front/main/app/styles/base/_body.scss?

Comment (weronika) [inline front/main/app/styles/component/design-system/_wikia-styleguide-overrides.scss]: definitions in this file don't override ds styles, but wikia-styleguide styles

**Pull request merged**